developer-docs-site/docs/concepts/resources.md

```diff
@@ -7,9 +7,9 @@ id: "resources"
 
 On Aptos, on-chain state is organized into resources and modules. These are then stored within the individual accounts. This is different from other blockchains, such as Ethereum, where each smart contract maintains their own storage space. Additionally, [event handles](./events.md) are defined within resources. See [Accounts](./accounts.md) for more details on accounts.
```

Comment:
`Additionally, [event handles](./events.md) are defined within resources.`
Not sure why events are even mentioned here (and V1), let's delete the sentence?